src/bda/plone/productshop/browser/templates/listing.pt

```diff
@@ -27,11 +27,13 @@
     <a href=""
        tal:attributes="href product/obj/absolute_url">
       <img src="" alt=""
+           i18n:attributes="alt"
```

Comment:
Is there any magic I don't know about? What is it good for to define i18n:attributes for an empty attribute when the entire tag gets replaced?